py-polars/polars/series/series.py

```diff
@@ -2961,7 +2961,7 @@ def extend(self, other: Series) -> Self:
                 raise
         return self
 
-    def filter(self, predicate: Series | list[bool]) -> Self:
+    def filter(self, predicate: Series | list[bool] | Expr) -> Series:
         """
         Filter elements by a boolean mask.
```

Comment:
Also add " or `pl.element()` expression that evaluates to a boolean mask" here?

Reply:
That description is just below in the `Parameters` section, which I think is sufficient.